sentinel-transport/sentinel-transport-simple-http/src/main/java/com/alibaba/csp/sentinel/transport/heartbeat/HeartbeatMessage.java

```diff
@@ -30,7 +31,9 @@
  * The message consists of key-value pair parameters.
  *
  * @author leyou
+ * @deprecated please use {@link HeartbeatMessageProvider#getHeartbeatMessage()} instead
  */
+@Deprecated
```

Comment:
Is it ok to mark as `@Deprecated`?

Reply:
Yes, since a new interface has been introduced.